nixos/modules/tasks/network-interfaces.nix

```diff
@@ -1327,20 +1327,13 @@ in
 
     # Capabilities won't work unless we have at-least a 4.3 Linux
```

Comment:
I guess you can get rid of this comment as well.

Reply:
Good catch, thanks!